modules/articles/client/controllers/articles.client.controller.js

```diff
@@ -63,22 +48,25 @@ angular.module('articles').controller('ArticlesController', ['$scope', '$statePa
       var article = $scope.article;
 
       article.$update(function () {
-        $location.path('articles/' + article._id);
+        $state.go('articles.view',{articleId: article._id});
       }, function (errorResponse) {
         $scope.error = errorResponse.data.message;
       });
     };
 
-    // Find a list of Articles
-    $scope.find = function () {
-      $scope.articles = Articles.query();
+    // Remove existing Article
+    $scope.remove = function () {
+      if (confirm('Are you sure you want to delete?')) {
+        $scope.article.$remove(function (article) {
+          if ($scope.articles !== undefined && $scope.articles.length > 0) {
+            $scope.articles.splice($scope.articles.indexOf($scope.article), 1);
+          }
+          $state.go('articles.list');
+        });
+      }
     };
 
-    // Find existing Article
-    $scope.findOne = function () {
-      $scope.article = Articles.get({
-        articleId: $stateParams.articleId
-      });
-    };
+    //Not sure if this is needed anymore?
```

Comment:
Yeah its needed to display edit/remove on the view article.  Would you all be opposed to do a scope variable canEdit canRemove instead of putting the whole Authentication object in scope and checking in the view?